core-bundle/assets/controllers/module-wizard-controller.js

```diff
@@ -0,0 +1,147 @@
+import {Controller} from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['body', 'rowTemplate', 'row'];
+
+    rowSnapshots = new Map();
+
+    rowTemplateTargetConnected(template) {
+        // We need to queue a micro task here, so that Stimulus will fire
+        // rowTargetConnected().
+        queueMicrotask(() => {this._unwrap(template)});
+    }
+
+    rowTargetConnected() {
+        this._makeSortable();
+    }
+
+    rowTargetDisconnected() {
```

Comment:
```suggestion
    rowTargetDisconnected(row) {
```

Reply:
Assets need to be rebuilt.

Reply:
Did that in 19f7e966d648532d02c7e4a0c4917f8c80359980 :)